python_modules/dagster/dagster/_grpc/types.py

```diff
@@ -292,25 +292,27 @@ class ListRepositoriesResponse(
             ("entry_point", Optional[Sequence[str]]),
             ("container_image", Optional[str]),
             ("container_context", Optional[Mapping[str, Any]]),
+            ("dagster_libraries", Optional[Mapping[str, str]]),
```

Comment:
consider dagster_library_versions or dagster_library_metadata or some noun? a naive person might think we had zipped up the whole package or something